recipes/elephas/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "elephas" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5cb28cb23827b45d96f5af59379cd7f739ee9e5785e9b61fe3a22440d4fea528
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - cython
```

Comment:
are you compiling some stuff in this recipe? If so, I think this cannot be ``noarch`` :(

Reply:
@marcelotrevisani I will keep cython and change the noarch